packages/grafana-ui/src/themes/mixins.ts

```diff
@@ -47,6 +47,13 @@ export function listItem(theme: GrafanaTheme): string {
     `;
 }
 
+export function listItemSelected(theme: GrafanaTheme): string {
+  return `
+       background: ${theme.isLight ? theme.colors.gray6 : theme.colors.dark9};
+       color: ${theme.colors.textStrong};
+    `;
+}
+
```

Comment:
stylesFactory please:)

Reply:
this is used in a other styles, it returns a string :)

Reply:
Nice, totally missed that. But I guess we shouldn't wrap the mixins in it, since those are used inside other styles, which in turn are wrapped in the factory.